src/components/Explore/Explore.tsx

```diff
@@ -64,8 +64,14 @@ const Explore: React.FunctionComponent<{
   ]);
 
   const onChangeSelectedLocations = (newLocations: Location[]) => {
+    const selectedLocations = uniq([currentLocation, ...newLocations]);
```

Comment:
Consider renaming this variable so it has a different name than the state variable `selectedLocations`. I know that the local variable will take precedence, but a different name will make that more obvious.